pkg/state/transaction_checker.go

```diff
@@ -49,14 +50,16 @@ type transactionChecker struct {
 	genesis  proto.BlockID
 	stor     *blockchainEntitiesStorage
 	settings *settings.BlockchainSettings
+	state    types.SmartState
 }
 
 func newTransactionChecker(
 	genesis proto.BlockID,
 	stor *blockchainEntitiesStorage,
 	settings *settings.BlockchainSettings,
+	state types.SmartState,
```

Comment:
Why pass SmartState if we already in state package?

Reply:
Removed. Now `txChecker` uses `stor *blockchainEntitiesStorage` instead of `state types.SmartState`.